recirq/qml_lfe/sample_analysis.ipynb

```diff
@@ -0,0 +1,1451 @@
+     "text": [
+      "LOSS 0.02484862251330469\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      " 83%|██████████████████████████████▊      | 2501/3000 [12:30<1:43:46, 12.48s/it]"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "ACC 1.0\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      " 87%|█████████████████████████████████▊     | 2601/3000 [12:40<00:45,  8.84it/s]"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "LOSS 0.04351241139576814\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      " 90%|███████████████████████████████████    | 2700/3000 [12:51<00:28, 10.46it/s]"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "LOSS 0.03283559548246558\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      " 92%|███████████████████████████████████▊   | 2751/3000 [13:49<31:55,  7.69s/it]"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "ACC 1.0\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      " 93%|████████████████████████████████████▍  | 2802/3000 [13:55<00:21,  9.22it/s]"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "LOSS 0.027417695932817877\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      " 97%|█████████████████████████████████████▋ | 2900/3000 [14:05<00:08, 11.27it/s]"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "LOSS 0.018936340993477643\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|██████████████████████████████████████▉| 2999/3000 [14:18<00:00,  4.79it/s]"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "LOSS 0.02224759492623889\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|███████████████████████████████████████| 3000/3000 [15:09<00:00,  3.30it/s]"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "ACC 1.0\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "from tqdm import tqdm\n",
+    "import cirq\n",
+    "import numpy as np\n",
+    "from os import listdir\n",
+    "import random\n",
+    "from os.path import isfile, join\n",
+    "from cirq import Simulator\n",
+    "simulator = Simulator()\n",
+    "\n",
+    "all_paulis = dict([(\"I\",0), (\"X\",1), (\"Y\",2), (\"Z\",3)])\n",
```

Comment:
Again, I feel like this code (or similar) is repeated all over.  We should have a standard lookup people can use.